plugins/extractors/bigquery/profile_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build plugins
+// +build plugins
+
```

Comment:
Added plugins tags for tableau and bigquery tests, because we test extractors separately.